docs/docs/reference/cli.md

```diff
@@ -370,13 +375,31 @@ talhelper gencommand upgrade [flags]
 
 ```
   -c, --config-file string    File containing configurations for talhelper (default "talconfig.yaml")
+      --context               Use clusterName from talconfig.yaml as talosctl context instead of --talosconfig flag.
   -d, --debug                 Whether to enable debugging mode
   -e, --env-file strings      List of files containing env variables for config file (default [talenv.yaml,talenv.sops.yaml,talenv.yml,talenv.sops.yml])
       --extra-flags strings   List of additional flags that will be injected into the generated commands.
   -n, --node string           A specific node to generate the command for. If not specified, will generate for all nodes.
   -o, --out-dir string        Directory that contains the generated config files to apply. (default "./clusterconfig")
 ```
 
+### Example
```

Comment:
This docs is autogenerated by `talhelper gendocs cli` subcommand, so this example will be gone when the CI runs.